internal/services/storage/storage_account_resource_test.go

```diff
@@ -888,6 +888,7 @@ func TestAccStorageAccount_replicationTypeGZRS(t *testing.T) {
 }
 
 func TestAccStorageAccount_largeFileShare(t *testing.T) {
+	// Note: this test causes a ForceNew when disabling large file shares
```

Comment:
This feels like we should probably just remove the test?

Reply:
Yeah good point, I'll remove the final step from the test